crates/api_common/src/post.rs

```diff
@@ -135,14 +135,15 @@ pub struct RemovePost {
   pub reason: Option<String>,
 }
 
+#[skip_serializing_none]
```

Comment:
Seems unnecessary for something that is only received, not sent.

Reply:
Its necessary for ts-rs.